homeassistant/components/automation/__init__.py

```diff
@@ -38,13 +38,15 @@
 CONF_ACTION = 'action'
 CONF_TRIGGER = 'trigger'
 CONF_CONDITION_TYPE = 'condition_type'
+CONF_INITIAL = 'initial'
```

Comment:
I agree that initial_state is better, but I wanted to make it consistent with input_slider, input_boolean, arduino switch and input_select.
Should I still change it to initial_state?